<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::MimeTypeSettingsPage</class>
 <widget class="QWidget" name="Core::Internal::MimeTypeSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="mimeTypesGroupBox">
      <property name="title">
       <string>Registered MIME Types</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="filterLineEdit">
         <property name="placeholderText">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="resetButton">
         <property name="toolTip">
          <string>Reset all to default.</string>
         </property>
         <property name="text">
          <string>Reset All</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>476</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTreeView" name="mimeTypesTreeView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="detailsGroupBox">
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="patternsLabel">
           <property name="text">
            <string>Patterns:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="patternsLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTreeWidget" name="magicHeadersTreeWidget">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>4</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">Magic Header</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Range</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Priority</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="addMagicButton">
             <property name="text">
              <string>Add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editMagicButton">
             <property name="text">
              <string>Edit...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeMagicButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>72</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
